bin/runtime/src/lib.rs

```diff
@@ -416,6 +435,51 @@ impl pallet_multisig::Config for Runtime {
     type WeightInfo = pallet_multisig::weights::SubstrateWeight<Runtime>;
 }
 
+// We do not burn any money within treasury.
+pub const TREASURY_BURN: u32 = 0;
+// The percentage of the amount of the proposal that the proposer should deposit.
+// We agreed on non-progressive deposit.
+pub const TREASURY_PROPOSAL_BOND: u32 = 0;
+// The proposer should deposit max{`TREASURY_PROPOSAL_BOND`% of the proposal value, $10}.
+pub const TREASURY_MINIMUM_BOND: Balance = 1000 * CENTS;
+// Every 4h we implement accepted proposals.
+pub const TREASURY_SPEND_PERIOD: BlockNumber = 4 * HOURS;
+// We allow at most 20 approvals in the queue at once.
+pub const TREASURY_MAX_APPROVALS: u32 = 20;
+
+parameter_types! {
+    pub const Burn: Permill = Permill::from_percent(TREASURY_BURN);
+    pub const ProposalBond: Permill = Permill::from_percent(TREASURY_PROPOSAL_BOND);
+	pub const ProposalBondMinimum: Balance = TREASURY_MINIMUM_BOND;
+    pub const MaxApprovals: u32 = TREASURY_MAX_APPROVALS;
+    pub const SpendPeriod: BlockNumber = TREASURY_SPEND_PERIOD;
+    pub const TreasuryPalletId: PalletId = PalletId(*b"py/trsry");
```

Comment:
What does this ID do?

Reply:
Oh, it's the ID of the associated account? Why is it called what it's called then, is there a standard here?

Reply:
Indeed, the `PalletId` is used to derive the corresponding account, an example in the pallet_treasury:
```rust
/// The treasury's pallet id, used for deriving its sovereign account ID.
#[pallet::constant]
type PalletId: Get<PalletId>;
```

In substrate these are some used names:
 - `PalletId(*b"py/trsry")`
 - `PalletId(*b"py/socie")`
 - `PalletId(*b"py/lotto")`

I followed their way. Also, afaik, it has to be of length 8

Reply:
> it has to be of length 8

...what? lol

In this case it cannot be improved much, but it would still be nice to know what the "py" refers to – it would be a bit silly if it was somehow supposed to be associated with e.g. Polkadot. Although the 'p' probably stands for "pallet"...?

Reply:
regarding length: `pub struct PalletId(pub [u8; 8]);` (https://github.com/paritytech/substrate/blob/4581dd99ed59f65ede46a34c81133177d5824a68/frame/support/src/lib.rs#L111)

Reply:
![image](https://user-images.githubusercontent.com/27450471/142602076-edbe69b0-8135-4a85-b4fc-7e8eea2142a1.png)

Reply:
I dug it up, and now I wonder whether e.g. `a0/trsry` would be better :P

Reply:
Excellent, I was afraid that it was something like this and well done finding such a direct proof. :D

"a0" as a prefix is indeed much better.

Reply:
renamed